sdk/template/template/recordings/browsers/core_v1_tests_using_unified_recorder/recording_storagequeue_create_queue.json

```diff
@@ -0,0 +1,42 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://account_name.queue.core.windows.net/queue-1326?sv=2020-08-04\u0026ss=bfqt\u0026srt=sco\u0026sp=rwdlacuptfx\u0026se=2026-07-10T07:00:24Z\u0026st=2021-07-09T23:00:24Z\u0026spr=https\u0026sig=fake_sig\u0026timeout=30",
```

Comment:
Wow the number of escaped things here 👀 I wonder what’s the underlying reason.

Reply:
This is [most likely coming from the .NET JSON serializer](https://docs.microsoft.com/en-us/dotnet/standard/serialization/system-text-json-how-to?pivots=dotnet-6-0#serialization-behavior). The default escaping behavior appears to be quite aggressive.

> The default encoder escapes non-ASCII characters, HTML-sensitive characters within the ASCII-range, and characters that must be escaped according to the RFC 8259 JSON spec.

Presumably `+` and `&` fall under "HTML-sensitive characters within the ASCII-range". It doesn't matter in practice -- the Unicode-escaped values will deserialize to the same thing as if the real characters were there.

Reply:
ok ok gotcha! As long as it’s expected, it looks good to me 😊